Marlin/Marlin_main.cpp

```diff
@@ -1715,18 +1722,10 @@ static void clean_up_after_endstop_or_probe_move() {
       z_dest -= zprobe_zoffset;
 
     if (z_dest > current_position[Z_AXIS]) {
-      float old_feedrate = feedrate;
-      feedrate = homing_feedrate[Z_AXIS];
       do_blocking_move_to_z(z_dest);
-      feedrate = old_feedrate;
```

Comment:
I think we need to still set `feedrate` here, because code that calls `do_probe_raise` isn't consistently setting the `feedrate` for a Z move.